packages/service-content/src/lib/resources/queries/get-newsletters.ts

```diff
@@ -0,0 +1,31 @@
+import { sql } from '@blms/database';
+import type { JoinedNewsletter } from '@blms/types';
+
+export const getNewslettersQuery = (language?: string) => {
```

Comment:
Changes to make there related to expected changes in drizzle schema (use language in localized table instead)